src/features/content-answers/ContentAnswers.tsx

```diff
@@ -4,10 +4,13 @@ import ContentAnswersModal from './ContentAnswersModal';
 import ContentAnswersOpenButton from './ContentAnswersOpenButton';
 
 type ExternalProps = {
```

Comment:
`ExternalProps` is used by the customer to pass specific props to configure ContentAnswers: https://github.com/box/box-ui-elements/blob/master/src/elements/content-preview/ContentPreview.js#L233

the new props like `fileExtension`, etc. can't be used by customer

Reply:
changed